### PR TITLE
Implement Login screen without backend and storybook integration

### LINKED ISSUE #2: Implement Login screen
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Login uses the backend to check the user's credentials (username, password) and forwards the user to another screen. The current user information is stored so that other components can access the user information.

**Describe alternatives you've considered**
None

**Additional context**
Login works with JWT access and refresh tokens. Need to be securely stored (SecureStorage).
Try to use Storybook for this component.

IMG_0346

### PR DESCRIPTION
**Pull request by DominicLuidold**
Closes #2

### CODE REVIEW

**Comment by DominicLuidold** [comment]
@alexw-at @whiteclouds7 Sorry for hijacking your issue/pull request. I'm currently trying something cool regarding automatic deployment of the app based on a PR and I needed something I can test this with.

// Edit
It appears that this will only work based on the latest changes which haven't been populated to your branch yet. Please feel free to keep working on #2 and edit the description of the PR.

File: src/services/i18n.ts [hunk 1]
```diff
@@ -3,22 +3,57 @@ import { initReactI18next } from "react-i18next";
...
+    de: {
         translation: {
-          "hello_world": "Hallo Welt",
+            login: "Anmelden",
+            register: "Registrieren",
+            email: "Email",
+            password: "Passwort",
+            repeatPassword: "Passwort Wiederholen",
+            username: "Benutzername",
+            startJourney: ["Resie Starten", "Los gehts"],
```
Comment (valentingc) [inline src/services/i18n.ts]: typo: "R**ei**se starten" instead of "Resie Starten"

**Review by valentingc — APPROVED** [review]
Just a minor typo, looks great!

File: src/screens/Register.tsx [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+    const [email, setEmail] = useState("");
+    const [password, setPassword] = useState("");
+    const [password2, setPassword2] = useState("");
+    const [username, setUsername] = useState("");
+
+    const handleRegister = () => {
+        console.log(t("startJourney", { returnObjects: true }));
+        if (password !== password2) {
+            return;
+        }
```
Comment (DominicLuidold) [inline src/screens/Register.tsx:18]: No visual clue when the two passwords don't equal is probably just a temporary thing, right?

**Review by DominicLuidold — COMMENTED** [review]
I only found this, and I am not sure if this is done on purpose because we need to get this PR ready or if this was an oversight.

**Comment by DominicLuidold** [comment]
@whiteclouds7 is the PR ready to merge from your side?

**Pull request merged**